livekit-agents/livekit/agents/stt/stt.py

```diff
@@ -39,13 +39,23 @@ class SpeechEvent:
     alternatives: List[SpeechData] = field(default_factory=list)
 
 
+@dataclass
+class STTCapabilities:
+    streaming_supported: bool
+    interim_results: bool
+
+
 class STT(ABC):
```

Comment:
I sometimes wonder if our STT should be an abstract superclass, so stuff like this could be more easily inherited.